rpmbuild/copr-rpmbuild.spec

```diff
@@ -103,7 +107,7 @@ Requires: dnf-yum
 Requires: dnf-utils
 %endif
 # selinux toolset to allow running ansible against the builder
-%if 0%{?fedora}
+%if 0%{?fedora} || 0%{?openEuler}
```

Comment:
I believe we can drop this `%if` entirely.

Reply:
Done